hack/schemas/main.go

```diff
@@ -171,14 +171,22 @@ func generateSchema(root string, dryRun bool, version schema.Version) (bool, err
 	return same, nil
 }
 
-func yamlFieldName(field *ast.Field) string {
```

Comment:
this returns yaml tag key, so i changed it to yamlTagValue, maybe yamlTagKey is more appropriate